cmake/RunTests.cmake

```diff
@@ -24,32 +24,37 @@ if(BUSTED_OUTPUT_TYPE STREQUAL junit)
   set(EXTRA_ARGS OUTPUT_FILE ${BUILD_DIR}/${TEST_TYPE}test-junit.xml)
 endif()
 
+set(BUSTED_ARGS $ENV{BUSTED_ARGS})
+separate_arguments(BUSTED_ARGS)
+
 if(DEFINED ENV{TEST_TAG} AND NOT "$ENV{TEST_TAG}" STREQUAL "")
-  set(TEST_TAG "--tags=$ENV{TEST_TAG}")
+  list(APPEND BUSTED_ARGS "--tags='$ENV{TEST_TAG}'")
 endif()
 
 if(DEFINED ENV{TEST_FILTER} AND NOT "$ENV{TEST_FILTER}" STREQUAL "")
-  set(TEST_FILTER "--filter=$ENV{TEST_FILTER}")
+  list(APPEND BUSTED_ARGS "--filter='$ENV{TEST_FILTER}'")
```

Comment:
Unless you are certain the single-quotes are needed I would leave them out. CMake generally adds the correct platform-specific quotations where needed, and adding quotes can cause surprising problems .

Reply:
It is needed - see the description of https://github.com/neovim/neovim/pull/10239/commits/e119c9ce682feb829761c1870d8e47387baac454.

Reply:
I would guess that single-quotes won't work on Windows if cmd.exe is involved somehow. Try this instead:

    list(APPEND BUSTED_ARGS --filter="$ENV{TEST_FILTER}")

Reply:
This appears to work as well, will use that.

Reply:
Actually, I bet it works without any quotes:

     list(APPEND BUSTED_ARGS --filter=$ENV{TEST_FILTER})

Reply:
Yeah, might be true given that is a list now.

Reply:
No.. check `TEST_FILE=test/functional/terminal/tui_spec.lua TEST_FILTER="TUI background color handles" make functionaltest`: